src/codegen.cpp

```diff
@@ -4102,6 +4191,49 @@ static LLVMValueRef ir_render_ctz(CodeGen *g, IrExecutable *executable, IrInstru
     return gen_widen_or_shorten(g, false, int_type, instruction->base.value.type, wrong_size_int);
 }
 
+static LLVMValueRef ir_render_shuffle_vector(CodeGen *g, IrExecutable *executable, IrInstructionShuffleVector *instruction) {
+    uint64_t len_a = instruction->a->value.type->data.vector.len,
+        len_c = instruction->mask->value.type->data.vector.len;
```

Comment:
For clarity I would make this an independent statement.

Reply:
Yeah